static/documentation/forking.md

```diff
@@ -0,0 +1,38 @@
+# Working with a forked repo
+
+## Forking the repo
+
+In order to contribute to this project you will need to create a fork of the repo on your own account. 
```

Comment:
Missing comma:
**In order to contribute to this project, you will need to create a fork of the repo on your own account.**

Reply:
Updated as suggested.